test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/jbpm/ProcessFailoverIntegrationTest.java

```diff
@@ -129,7 +131,8 @@ public void processFailoverTest() {
         processServicesClient.signalProcessInstance(CONTAINER_ID, longScriptPid, Constants.Signal.SIGNAL_2_NAME, null);
         assertProcessInstanceState(longScriptPid, org.kie.api.runtime.process.ProcessInstance.STATE_COMPLETED);
 
-        assertThat(deploymentScenario.getKieServerDeployment().getInstances().iterator().next()).isNotEqualTo(kieServerInstance);
+        Collection<String> kieServerNewInstances = getKieServerInstanceNames();
+        kieServerOldInstances.forEach(assertThat(kieServerNewInstances)::doesNotContain);
```

Comment:
We can simplified it and remove the for each. I think we could be able to run assert on collection so it would look like this: `assertThat(kieServerOldInstance).doesNotContaine(kieServerNewInstances);`

Reply:
Indeed, it looks better with:

```java
assertThat(kieServerOldInstances).doesNotContainAnyElementsOf(getKieServerInstanceNames());
```